ref/general/network-hardening.adoc

```diff
@@ -0,0 +1,223 @@
+// Copyright (c) 2020 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description:
+:seo-title: Network security hardening
+:seo-description:
+:page-layout: general-reference
+:page-type: general
+= Network security hardening
+
+Network intrusions occur when unauthorized users gain access to network activity.
+With this access, they can alter traffic and steal network resources.
+
+// General questions
+// What are the most common examples of network intrusions or attacks? Would this be helpful in the intro?
+// What's the advice for where to put things in the cloud? Is this determined by the cloud or do you have to decide for yourself?
+// Is any of this information irrelevant in a cloud-native microservices context? 
+// Are there any gaps that I've missed?
+
+== Run behind firewalls
+Your production environment needs to run inside a firewall so your applications don't run in the DMZ.
+The DMZ is an intermediary zone where proxy and web servers run that is located between public and private networking zones.
+Typically, your network setup includes two firewalls, a perimeter firewall and a backend firewall.
+The perimeter firewall exists between the public zone and the DMZ, and the backend firewall exists between the DMZ and the private zone, as shown in the following topology diagram:
+
+<diagram pending>
+
+// Do I need to include the information from the slide about IHS?
+
+== Enable SSL connectivity
+Open Liberty provides a simple SSL configuration with the link:/docs/ref/feature/#transportSecurity.html[Transport Security feature].
+The Transport Security feature enables the link:/docs/ref/feature/#ssl.html[SSL feature] to support SSL connections and adds the ability to configure outbound transports.
+SSL uses signed public keys to protect HTTP, IIOP, LDAP, MQ, JDBC, and messaging traffic between Open Liberty messaging engines, J2C, and SOAP traffic.
+The default SSL configuration is simple and easy to configure, but it is not suitable for use in production because it generates a self-signed certificate.
```

Comment:
When describing the SSL configuration as "simple and easy to configure", I think it might be good to add a point of comparison or additional reasoning to provide more context. Why is it simple and easy to configure?  I think adding a bit more detail here could help frame the topic around what's being improved or modified.